code/datums/namepool.dm

```diff
@@ -35,3 +35,9 @@ GLOBAL_LIST_EMPTY_TYPED(namepool, /datum/namepool)
 	firstname_male_pool = "names/moth_first"
 	firstname_female_pool = "names/moth_first"
 	lastname_pool = "names/moth_last"
+
+/datum/namepool/sectoid
+	lastname_pool = "names/sectoid_last"
+
+/datum/namepool/sectoid/get_random_name()
+	return "Sectoid [pick(SSstrings.get_list_from_file(lastname_pool))]"
```

Comment:
```suggestion
	return "Sectoid [pick(ascii2text(rand(49, 57)))][pick(ascii2text(rand(65, 90)))][pick(ascii2text(rand(65, 90)))]"
```
65 to 90 is A to Z (uppercase) in ASCII, and 49 to 57 is 1 to 9.
Same result as before, more variation and less chance of a repeated name.
You can use defines if you think it's not very clear, stuff like this:
```cpp
#define RAND_1_TO_9 (ascii2text(rand(48, 57)))
#define RAND_A_TO_Z (ascii2text(rand(65, 90)))
return "Sectoid [RAND_1_TO_9][RAND_A_TO_Z][RAND_A_TO_Z]"
```